tutorials/physics/ray-casting.rst

```diff
@@ -191,7 +192,7 @@ from a CharacterBody2D or any other collision object node:
 
     func _physics_process(delta):
         var space_state = get_world_2d().direct_space_state
-        var query = PhysicsRayQueryParameters2D.create(global_position, enemy_position)
+        var query = PhysicsRayQueryParameters2D.create(global_position, player_position)
```

Comment:
What's the reason for using `player_position` here but `target_position` below?

Reply:
the tutorial provides a specific-use case example (enemy & player), so i thought it would be a good idea to use more 'generalized' wording there to hint at other use-cases outside of those situations.